cypress/support/pickRandomly.js

```diff
@@ -0,0 +1,5 @@
+const pickRandomly = (array) => {
+  return array.filter(() => Math.random() > 0.5);
```

Comment:
The current implementation of `pickRandomly` returns a filtered array where each element has a 50% chance of being included. If the intention is to pick a single random element from the array, consider using a different approach, such as selecting a random index.